storage/engine/keys.go

```diff
@@ -372,6 +377,8 @@ var (
 	KeyLocalRaftTruncatedStateSuffix = proto.Key("rftt")
 	// KeyLocalRaftAppliedIndexSuffix is the suffix for the raft applied index.
 	KeyLocalRaftAppliedIndexSuffix = proto.Key("rfta")
+	// KeyLocalRaftLeaderLeaseSuffix is the suffix for the raft leader lease.
+	KeyLocalRaftLeaderLeaseSuffix = proto.Key("rfll")
```

Comment:
We should alphabetize these by value so it's easier to spot collisions. (I was going to say that all the existing raft suffixes start with `rft`, but `rftl` is already taken so we can't use it).